windows/include/unistd.h

```diff
@@ -0,0 +1,14 @@
+/* This file contains a unistd.h replacement for all 
+	* that is needed for lou_compare to work in windows.
+	* It is used so that the code in lou_compare is cleaner.
+	* Please add any more functionality here for future development.
+	* Author Paul Wood (Torch Trust)
+*/
```

Comment:
Probably should add some licensing info here, like "distributed under MIT license" like dirent.h above